src/test/java/org/assertj/core/api/classes/ClassAssert_hasPackage_Package_Test.java

```diff
@@ -0,0 +1,28 @@
+package org.assertj.core.api.classes;
+
+import static org.mockito.Mockito.verify;
+
+import org.assertj.core.api.ClassAssert;
+import org.assertj.core.api.ClassAssertBaseTest;
+import org.junit.jupiter.api.DisplayName;
+
+/**
+ * Tests for <code>{@link ClassAssert#hasPackage(Package)}</code>.
+ *
+ * @author Matteo Mirk
+ */
+@DisplayName("Classes hasPackage")
+class ClassAssert_hasPackage_Package_Test extends ClassAssertBaseTest {
+
+  static final Package PACKAGE = Package.getPackage("org.assertj.core.api");
```

Comment:
As `Package.getPackage()` is deprecated from java 9 onwards, I'd just use a mock here:
```suggestion
  static final Package PACKAGE = mock(Package.class);
```

Reply:
I see, since we're just testing internal interactions, this way the code will be future-proof when it's migrated past Java 8?